tests/nn/test_tracking_integration.py

```
@@ -146,6 +146,11 @@ def main(f, dir):
         0.25,
     )
 
+    kalman_params = dict(
```

Ruff (0.8.2)

Local variable `kalman_params` is assigned to but never used

Remove assignment to unused variable `kalman_params`

(F841)